compound/src/main/kotlin/io/element/android/compound/tokens/generated/internal/DarkColorTokens.kt

```diff
@@ -30,315 +30,315 @@ import io.element.android.compound.annotations.CoreColorToken
+    val colorAlphaCyan1000 = Color(0xe000bfe0)
+    val colorAlphaCyan1100 = Color(0xc926e7fd)
+    val colorAlphaCyan1200 = Color(0xd98af1ff)
+    val colorAlphaCyan1300 = Color(0xebc9f7fd)
+    val colorAlphaCyan1400 = Color(0xf5e1fbfe)
+    val colorAlphaBlue100 = Color(0xff00055c)
+    val colorAlphaBlue200 = Color(0xff00095c)
+    val colorAlphaBlue300 = Color(0xff001366)
+    val colorAlphaBlue400 = Color(0xff001e70)
+    val colorAlphaBlue500 = Color(0xa1003cbd)
+    val colorAlphaBlue600 = Color(0x87015afe)
+    val colorAlphaBlue700 = Color(0xa30665fe)
+    val colorAlphaBlue800 = Color(0xd61077fe)
+    val colorAlphaBlue900 = Color(0xeb4491fd)
+    val colorAlphaBlue1000 = Color(0xf062a0fe)
+    val colorAlphaBlue1100 = Color(0xf57cb2fd)
+    val colorAlphaBlue1200 = Color(0xf7a3c8ff)
+    val colorAlphaBlue1300 = Color(0xfccde1fe)
+    val colorAlphaBlue1400 = Color(0xffe6effe)
+    val colorAlphaPurple100 = Color(0xff1a0057)
+    val colorAlphaPurple200 = Color(0xff1d005c)
+    val colorAlphaPurple300 = Color(0xff22006b)
+    val colorAlphaPurple400 = Color(0xff2d0080)
+    val colorAlphaPurple500 = Color(0xff3d009e)
+    val colorAlphaPurple600 = Color(0xab690dfd)
+    val colorAlphaPurple700 = Color(0xc2712bfd)
+    val colorAlphaPurple800 = Color(0xeb7f4dff)
+    val colorAlphaPurple900 = Color(0xfa9271fe)
+    val colorAlphaPurple1000 = Color(0xfca28bfe)
+    val colorAlphaPurple1100 = Color(0xffab9afe)
+    val colorAlphaPurple1200 = Color(0xffc7bdff)
+    val colorAlphaPurple1300 = Color(0xffdfdbff)
+    val colorAlphaPurple1400 = Color(0xffeeebff)
+    val colorAlphaFuchsia100 = Color(0xff28003d)
+    val colorAlphaFuchsia200 = Color(0xff2d0042)
+    val colorAlphaFuchsia300 = Color(0xff36004d)
+    val colorAlphaFuchsia400 = Color(0xff45005c)
+    val colorAlphaFuchsia500 = Color(0x61ca0aff)
+    val colorAlphaFuchsia600 = Color(0x70d21fff)
+    val colorAlphaFuchsia700 = Color(0x8ad82ffe)
+    val colorAlphaFuchsia800 = Color(0xb5eb44fd)
+    val colorAlphaFuchsia900 = Color(0xccf172fd)
+    val colorAlphaFuchsia1000 = Color(0xd4f790fe)
+    val colorAlphaFuchsia1100 = Color(0xdbfaa4fe)
+    val colorAlphaFuchsia1200 = Color(0xe8fac3fe)
+    val colorAlphaFuchsia1300 = Color(0xf2fde0ff)
+    val colorAlphaFuchsia1400 = Color(0xfafdecfe)
+    val colorAlphaPink100 = Color(0xff38000f)
+    val colorAlphaPink200 = Color(0xff3d0012)
+    val colorAlphaPink300 = Color(0xff470019)
+    val colorAlphaPink400 = Color(0xff570024)
+    val colorAlphaPink500 = Color(0xff6b0036)
+    val colorAlphaPink600 = Color(0x75fb0473)
+    val colorAlphaPink700 = Color(0x94fd1277)
+    val colorAlphaPink800 = Color(0xccfe1b79)
+    val colorAlphaPink900 = Color(0xf5fe4382)
+    val colorAlphaPink1000 = Color(0xfaff6691)
+    val colorAlphaPink1100 = Color(0xfffe86a4)
+    val colorAlphaPink1200 = Color(0xffffadc0)
+    val colorAlphaPink1300 = Color(0xffffd1db)
+    val colorAlphaPink1400 = Color(0xffffebef)
```

Comment:
A shame that this cannot be diff easily. We should probably sort the `val` alphabetically.

Reply:
Maybe there's something we can add in https://github.com/element-hq/compound-design-tokens for that, but I somehow doubt it'll be that easy.